New unknown shuttle: Gas Freighter

a-random-lemurian opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Unknown shuttle gas freighter.

This is my first pull request. Hello.

The gas freighter isn't ready yet, but I got good feedback in #cartography when I posted a screenshot of it and decided to open this PR to get more feedback and see if I can get my gas freighter into vanilla, of course after improving it and polishing things a little.

### Issues
- [ ] Rotated the wrong way: helm is west-facing, instead of north-facing.
- [x] Undecided name. The naming format of ships like "NT-Example" really works only with 1-word names, but the freighter is called the "Plasma Queen". Maybe a better one-word name works better.
  - Turns out the format is NT "Multi-word Name", with NT-Single only for one-word names.
- [x] I haven't actually linked those air alarms to the sensors yet.
- [x] I don't know what floor tiles to use for the ship.
- [ ] Should I add one tritium canister?

## Media
![image](https://github.com/user-attachments/assets/f0b440aa-1a59-4467-a33c-41ecabf82812)
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Lemuria
- add: Added a gas freighter unknown shuttle.

IProduceWidgets reviewed (changes requested):
Needs an entry in Resources/Prototypes/GameRules/unknown_shuttles.yml

a-random-lemurian commented:
> Needs an entry in Resources/Prototypes/GameRules/unknown_shuttles.yml

I think I've already added the entry.

https://github.com/space-wizards/space-station-14/pull/32850/files/fdbf3e6c22e5d06c83c9635fdc76e206a34e7408#diff-078cac60675a351ba51017d75d259be403556bfb6c852340226edb047496a182

a-random-lemurian commented:
Wonder which one I should go for..

## 1 
![image](https://github.com/user-attachments/assets/dc315d3f-e3af-4203-a674-14b8b7f6d8a4)

## 2
![image](https://github.com/user-attachments/assets/f65f8702-339b-48fc-adbd-a24746541cae)

IProduceWidgets commented:
> > Needs an entry in Resources/Prototypes/GameRules/unknown_shuttles.yml
> 
> I think I've already added the entry.
> 
> https://github.com/space-wizards/space-station-14/pull/32850/files/fdbf3e6c22e5d06c83c9635fdc76e206a34e7408#diff-078cac60675a351ba51017d75d259be403556bfb6c852340226edb047496a182

My apologies, I copied the wrong file path: Resources/Prototypes/Shuttles/shuttle_incoming_event.yml

YotaXP commented:
Heck yeah, more engineering visitors please.

> Wonder which one I should go for..

The first one feels closer to that of an engineer ship bridge. Function over form. Maybe throw in some racks and closets.

- Three SMES and two substations feels a little excessive.
- A small amount of steel (even just a 10-stack) would be appreciated. The first thing I would like to do on this shuttle is add volume pumps before the extraction connectors.
- Regarding the tritium canister, spawn one in and see what it appraises for. I suspect it wouldn't have an especially sell high value since it's at room temperature.
- I find it funny that this atmospherics shuttle has no atmospherics. There's plenty of space in the back there for air canisters.
- Also consider portable scrubbers and space heaters.

a-random-lemurian commented:
> Heck yeah, more engineering visitors please.
Indeed.

> > Wonder which one I should go for..
> 
> The first one feels closer to that of an engineer ship bridge. Function over form. Maybe throw in some racks and closets.

Yup, went with the first one.

> * Three SMES and two substations feels a little excessive.

Downscaled to one SMES, and one substation.

> * A small amount of steel (even just a 10-stack) would be appreciated. The first thing I would like to do on this shuttle is add volume pumps before the extraction connectors.

Full 30-stack.

> * Regarding the tritium canister, spawn one in and see what it appraises for. I suspect it wouldn't have an especially sell high value since it's at room temperature.

$4.8k

> * I find it funny that this atmospherics shuttle has no atmospherics. There's plenty of space in the back there for air canisters.

Haha, added the atmospherics.

> * Also consider portable scrubbers and space heaters.

Added space heaters but no room for scrubbers. Though I'd say the Plasma Queen is more for *transporting* gas, not managing a station's gas.

YotaXP commented:
- Waste appears to have no output. A passive vent out back would suffice.
- Air alarms should be rotated to face the room they'd be viewed from.
- Make sure the sole generator there can keep up with the passive power draw of the thrusters and other devices on board. Don't want the SMES to run dry while idle.
- Looks like the aft gyro is off. Are two really needed?

Honestly tho, this is just a visitor shuttle, so it doesn't need to be held to the same standards as a station.

IProduceWidgets commented:
Need to color the pipenets too, and I would add some more tiles and decaling to the aft room. Maybe some catwalks in there.

You can color the pipe networks with the command
> colornetwork [pipe entID] Pipe [hex color]

the usual colors are 
- #0055CCFF for distro
- #FF1212FF for waste

Merrokitsune commented:
Would “NT Phoron” work?

a-random-lemurian commented:
> Would “NT Phoron” work?

Hmm, NT-Phoron sounds like an OK name. Changing it

Alpha-Two commented:
Some additional comments:

1. Why are 6/9 canisters empty? It would make sense for at least half of them to be full.
![image](https://github.com/user-attachments/assets/3ffb409d-ebeb-4e3d-bc71-a74fdfc4c167)

2. Your shuttle has no ghost roles, perhaps 1 Atmos Tech and 1 Engineer should work, use the `visiting X spawner` prototypes available when mapping.

metalgearsloth commented:
Unfortunately preloaded grids are frozen pending map streaming but welcome to PR it once that is done / the freeze is lifted.
